### PR TITLE
add footer with links

### PR DESCRIPTION
**Pull request by gitboej**
in this PR i created a footer with some icons in the style of a navbar

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -47,3 +51,20 @@
...
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #ccc;
+  border-top: #000 solid;
+  padding: 5px;
+}
+
+a > img {
```
Comment (t0bbl) [inline styles.css:67]: i like it !

Comment (t0bbl) [reply styles.css:67]: ups. should have been a single comment

**Review by aaaleeenaaa — APPROVED** [review]
Great work!

**Review by t0bbl — APPROVED** [review]

**Review by Roland-Hufnagel — APPROVED** [review]
This looks fine to me

**Pull request merged**